### ISSUE TITLE
Don't use this plugin!

### ISSUE THREAD

**Issue by nicolomanni** [vaenow/cordova-plugin-app-update#146]
This morning we received an email directly from Google Play with this content

![immagine](https://user-images.githubusercontent.com/35760794/89985627-ac6af400-dc7b-11ea-8e1a-2a7046d42741.png)

Better not to use this plugin anymore.

**Comment by memanoj** [comment]
Yes I also face same issue and app is removed from the store don't use this plugin now ionic should have to remove this from their documentation as soon as possible

**Comment by carlo318** [comment]
Also an app I develop has been removed from the store for the same reason. 
But I still need this plugin to autoupdate some custom devices running my app. I will install the App directly from a private Server and autoupdate from there.
So the plugin is still very useful.

**Comment by kolbasa** [comment]
Please use this, I need some stars :sweat_smile:
https://github.com/kolbasa/cordova-plugin-apkupdater

![grafik](https://user-images.githubusercontent.com/3901582/92164406-99d87a80-ee35-11ea-87d2-8a8d0d5dcf32.png)

**Comment by nagthgr8** [comment]
Wish I should have stepped in here before installing the plugin, it's too late my App is removed from the play store, not sure if they are going to listen to Appeals. Ionic should not have recommended this plugin for its users when such a violation is caught by bots why not caught during the build process, is that intentional? Very pathetic for App developers

**Comment by heliomarpm** [comment]
Lucky for me, I found this link, I was already implementing it.
And as for cordova-plugin-code-push, will it also be blocked?

**Comment by rhiroshi** [comment]
why do you guys need a plugin to update your app if you are already using play store?

**Comment by deborahjames** [comment]
> Lucky for me, I found this link, I was already implementing it.
> And as for cordova-plugin-code-push, will it also be blocked?

Hey can anyone here tell what worked for them . I need to implement auto app  update in ionic capacitor . Thanks in advance.

**Comment by mhartington** [comment]
**Shameless plug**

Hey folks, not the plugin author, but if you're looking for alternatives that do not violate any google/apple app store policies, perhaps checkout Appflow.


https://ionic.io/appflow

**Comment by andreszs** [comment]
Apparently the best option is to use the **In-App Update API** from [this cordova plugin](https://github.com/andreszs/cordova-plugin-in-app-update) which allows both Flexible and Immediate updates from Play Store.

**Comment by cyril-tl** [comment]
@andreszs can I use it to get the apk from my own server ?
